equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/model/data/Crm14PSEMessagesModel.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.model.data;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Immutable;
+
+import java.util.Date;
+
+@Immutable @Entity
+@Table(name = "CrmFormCRM14PSEDetails")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Crm14PSEMessagesModel {
```

Comment:
Ideally these model should be named after the table. That makes it easier to keep track of what it refers to. Even without the need to look at the code.
But if we are keeping this name (which I like too), please change to `Crm14PSEMessageModel` because this model only holds 1 instance of message. It is the list used on the upper class the one holding multiple messages

Reply:
done